Editor/Resources/EditorWindow/GameLiftPlugin.uxml

```diff
@@ -12,21 +12,20 @@
             <ui:VisualElement class="Divider" />
             <ui:Button text="Help &amp; Documentation" name="Help" class="TabButton" />
         </ui:VisualElement>
-        <ui:ScrollView name="Content" horizontal-scroller-visibility="Hidden" class="main__content">
-            <ui:VisualElement name="LandingTab" class="TabContent">
-                <ui:Label text="Landing" />
-            </ui:VisualElement>
-            <ui:VisualElement name="CredentialsTab" class="TabContent">
-                <ui:Label text="Credentials" />
-            </ui:VisualElement>
-            <ui:VisualElement name="AnywhereTab" class="TabContent">
-                <ui:Label text="Anywhere" />
-            </ui:VisualElement>
-            <ui:VisualElement name="EC2Tab" class="TabContent">
-                <ui:Label text="Managed EC2" />
-            </ui:VisualElement>
-            <ui:VisualElement name="HelpTab" class="TabContent">
-                <ui:Label text="Help" />
+        <ui:ScrollView name="Content" class="main__content">
+            <ui:VisualElement>
```

Comment:
Why are we adding an extra layer of nesting and removing the Landing item?

Reply:
Removed!